Library/Homebrew/test/cask/cmd/fetch_spec.rb

```diff
@@ -1,60 +0,0 @@
-# typed: false
-# frozen_string_literal: true
-
-describe Cask::Cmd::Fetch, :cask do
-  let(:local_transmission) do
-    Cask::CaskLoader.load(cask_path("local-transmission"))
-  end
-
-  let(:local_caffeine) do
-    Cask::CaskLoader.load(cask_path("local-caffeine"))
-  end
-
-  it "allows downloading the installer of a Cask" do
-    transmission_location = CurlDownloadStrategy.new(
-      local_transmission.url.to_s, local_transmission.token, local_transmission.version,
-      cache: Cask::Cache.path, **local_transmission.url.specs
-    ).cached_location
-    caffeine_location = CurlDownloadStrategy.new(
-      local_caffeine.url.to_s, local_caffeine.token, local_caffeine.version,
-      cache: Cask::Cache.path, **local_caffeine.url.specs
-    ).cached_location
-
-    expect(transmission_location).not_to exist
-    expect(caffeine_location).not_to exist
-
-    described_class.run("local-transmission", "local-caffeine")
-
-    expect(transmission_location).to exist
-    expect(caffeine_location).to exist
-  end
-
-  it "prevents double fetch (without nuking existing installation)" do
```

Comment:
We don't actually have a test for this and the one below for `brew fetch`. Would be good to preserve these two.